app/modules/utils.py

```diff
@@ -52,30 +52,21 @@ def is_valid_longitude(value):
     return min_longitude <= value <= max_longitude
 
 
-# this is a messy one. do we want to require timezone?  do we accept
-#   anything other than "full" ISO-8601?  for now just taking only full
-#   date + time (seconds optional) and ignoring timezone.  ymmv / caveat emptor / etc
+# Make this quite robust. Timezone is mandatory. Anything else is just making our life
```

Comment:
is this comment a TODO?

Reply:
It's not, but I can see why you think it is, will reword it.